### ISSUE TITLE
No comfy area

### ISSUE THREAD

**Issue by l-2-j** [neos-modding-group/world#3]
There should be a comfy area to just chill at:

- [ ] Mirrors, with local only toggles.
- [ ] Cheese. But make sure that it only appears when it's required.
- [ ] Good lighting, preferably brightness & temperature can be changed
- [ ] Sofa / bed / carpet kinda soft things to rest on
- [ ] Sitting anchors for desktop dwellers

**Comment by rampa3** [comment]
Yes! Cheese for Prime! Should be rigged to only appear when Prime is in the session maybe? XD
>     * [ ]  ....cheese?
Definitely mirrors, but should be togglable - Neosers usually don't have mirror rooms as hangout places.  And I don't mean normal mirror shutdown, but disabling the object itself.
>     * [ ]  .... I'm running out of mirror ideas, but you get the point. VRC refugees need mirrors x'P
How about dimmable with customizable light temperature? (Cool white ~ Warm white)
>     * [ ]  Not overly bright lighting, but not too dim either
Definitely, and don't forget sitting anchors for desktop dwellers!
>     * [ ]  Sofa / bed / carpet kinda soft things to rest on?
